pkg/machines/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -175,9 +180,9 @@ const StoragePoolSourceRow = ({ onValueChanged, dialogValues }) => {
                     <input id='storage-pool-dialog-source'
                            type='text'
                            minLength={1}
-                           placeholder={_("The directory on the server being exported")}
-                           value={dialogValues.source.dir || ''}
-                           onChange={e => onValueChanged('source', {'dir': e.target.value})}
+                           placeholder={dialogValues.type == 'netfs' ? _("The directory on the server being exported") : _("iSCSI IQN")}
```

Comment:
Maybe it's clear for someone who uses iSCSI, but could "IQN" perhaps get a tooltip to explain what that means?

Reply:
It's the iSCSI target identifier. Target for iscsi is the iscsi server. I changed the placeholder to be ISCSI target IQN. (To not be confused with initiator IQN which is the client identifier).
Do you think that this is ok or tooltip is still needed?

Reply:
Fine for me, thanks! "iscsi iqn" is also trivial to google, for people who are not familiar with it, and it seems to be common terminology for iSCSI.